components/learn/modules/ROOT/pages/deploying-and-interacting.adoc

```diff
@@ -138,13 +138,13 @@ module.exports = {
 ```
 --
 
-[.buidler]
+[.hardhat]
 --
-We need to add in our https://buidler.dev/config[configuration] that we are using the `buidler-ethers` plugin.
+We need to add in our https://hardhat.org/config[configuration] that we are using the `hardhat-ethers` plugin.
 
 ```js
-// buidler.config.js
-usePlugin('@nomiclabs/buidler-ethers');
+// hardhat.config.js
+require('@nomiclabs/hardhat-ethers');
```

Comment:
You just have to require/import their npm package to load a plugin now (instead of using the usePlugin function) https://hardhat.org/guides/migrate-from-buidler.html

Reply:
Yes this works by using `require` now.